src/client/createSqlmancerClient.ts

```diff
@@ -38,7 +38,8 @@ export type GenericSqlmancerClient = Knex & {
 
 export function createSqlmancerClient<T extends GenericSqlmancerClient = GenericSqlmancerClient>(
   glob: string,
-  knex: Knex
+  knex: Knex,
+  pubSub?: any
```

Comment:
```suggestion
import { PubSub } from 'graphql-subscriptions'

pubSub: PubSub
```

In TypeScript, [classes represent both types and values](https://www.typescriptlang.org/docs/handbook/classes.html#constructor-functions). `PubSub` describes both the type of an instance of PubSub and the underlying ES class.